docs/sources/next/javascript-api/jslib/http-instrumentation-pyroscope/client.md

```diff
@@ -0,0 +1,74 @@
+---
+title: 'Client'
+description: 'Client is a HTTP client attaching tracing information to its requests.'
+weight: 02
+---
+
+# Client
+
+`Client` is an HTTP client constructor that attaches baggage header to its requests. Use it to include a context in HTTP requests so that  [Grafana pyroscope](https://grafana.com/oss/pyroscope/) can incorporate their results.
+
+The `Client` class acts as a drop-in replacement for the standard `http` module and attaches a [baggage header](https://www.w3.org/TR/baggage/) to the request. For details about propagation, refer to [About baggage header](https://grafana.com/docs/k6/<K6_VERSION>/javascript-api/jslib/http-instrumentation-pyroscope#about-baggage-header).
+
+The `Client` constructor accepts a function taking the method, body and params of each request and returning a map of headers to be added to the request. By default it appends baggage header with info necessary for Grafana Cloud Profiling to integrate with Grafana K6 Ccloud.
+
+For details about propagation, refer to [About baggage header](https://grafana.com/docs/k6/<K6_VERSION>/javascript-api/jslib/http-instrumentation-pyroscope#about-baggage-header).
+
+## Example
+
+This example demonstrates how to instantiate a client and use it to instrument HTTP calls. It also illustrates how you can use it alongside the standard `http` module to perform non-instrumented HTTP calls.
```

Comment:
The second sentence "It" has an antecedent of HTTP calls. Should this be example and not the call? However, the next "it" (use it alongside) isn't clear. Is this referring to the client or the example? Please feel free to edit this suggestion as needed. 
```suggestion
This example demonstrates how to instantiate a client and use it to instrument HTTP calls. The example also illustrates how you can use the client alongside the standard `http` module to perform non-instrumented HTTP calls.
```